srcpkgs/glibc/template

```diff
@@ -10,38 +10,6 @@ homepage="http://www.gnu.org/software/libc"
 distfiles="${GNU_SITE}/glibc/glibc-${version}.tar.xz"
 checksum=f3eeb8d57e25ca9fc13c2af3dae97754f9f643bc69229546828e3a240e2af04b
 patch_args="-Np1"
-# Do not strip these files, objcopy errors out.
-nostrip_files="
-	XBS5_ILP32_OFFBIG
-	XBS5_ILP32_OFF32
-	POSIX_V6_ILP32_OFF32
-	POSIX_V6_ILP32_OFFBIG
-	POSIX_V7_ILP32_OFF32
-	POSIX_V7_ILP32_OFFBIG
-	POSIX_V6_LP64_OFF64
-	POSIX_V7_LP64_OFF64
-	XBS5_LP64_OFF64
```

Comment:
are those files?

Reply:
they are shared objects in `/usr/lib/gconv` and `/usr/lib/getconf` not sure what they are for exactly.

Reply:
But xbps-src doesn't scan for symbols, only for files.

Reply:
Okay on second look there are executables in /usr/lib/getconf that match these filenames.

Reply:
ah!